deployment/upgradeLive.sh

```diff
@@ -11,7 +11,6 @@ if [ ! -d "$1" ] ; then
         echo try to unzip v$1.tar.gz
         tar -xvzf v$1.tar.gz
         mv liveDVR-$1 $1
-        bash get_bins "$1"
         cd $1
         nvm install
         cp /opt/kaltura/liveController/latest/common/config/configMapping.json /opt/kaltura/liveController/$1/common/config/
```

Comment:
We should maintain a configMapping file in the general liveController folder that will enable any version to take the correct configuration without relying on a previous one to exists.
so should be:
cp /opt/kaltura/liveController/configMapping.json /opt/kaltura/liveController/$1/common/config/